tests/model_hub_tests/torch_tests/hf_transformers_models

```diff
@@ -104,8 +104,9 @@ google/mobilebert-uncased,mobilebert
 google/mobilenet_v1_0.75_192,mobilenet_v1
 google/mt5-base,mt5
 google/owlvit-base-patch32,owlvit
-google/pix2struct-docvqa-base,pix2struct,skip,Load problem
-google/realm-orqa-nq-openqa,realm,skip,Load problem
+google/pix2struct-docvqa-base,pix2struct,xfail,Convert error: no conversion rule found for operations: aten::prod. Conversion is failed for: ov::align_types
+google/pix2struct-ai2d-base,pix2struct,xfail,Convert error: no conversion rule found for operations: aten::prod. Conversion is failed for: ov::align_types
```

Comment:
Why you add a model here? Is it different in any useful way?

Reply:
By the first link despite there was a lot of downloads there was not enough info how to load this model. Because of that i decided to add this model to the list as well, but i agree, they do not differ much. Left only one with greater number of downloads.